examples/ObjectDataExample.js

```diff
@@ -9,109 +9,78 @@ const { DateCell, ImageCell, LinkCell, TextCell } = require('./helpers/cells');
 const { Table, Column, Cell } = require('fixed-data-table-2');
 const React = require('react');
 
```

Comment:
You can ignore changes in this file.  I accidentally committed when working on #268.  Reverting now